src/network/graphClient.js

```diff
@@ -11,7 +11,9 @@ const httpLink = new HttpLink({
   uri: Routes.graphql.http,
 });
 
-const wsLink = new WebSocketLink({
+// the check for process.browser makes sure our tests run properly since
+// this can only be run in the browser. Source: http://tinyurl.com/ycsjfq6p
+const wsLink = process.browser && new WebSocketLink({
```

Comment:
according to the source you posted, it should be `process.browser ? ...`

Reply:
can you explain what the issue of having this check is for in relation to testing? if this is just for the dashboard tests, why would the graphClient need to be changed? the sortOrder test does not use the graphClient?

Reply:
1. when you're using a ternary, in most cases, if you're returning `null`, it's not necessary to use a ternary. The `&&` does essentially the same thing.
```
const wsLink = process.browser && whatever // "undefined" if no in browser
const wsLink = process.browser ? whatever : null // "null" if no browser
```

2. I'm honestly not quite sure why this is happening. From what I read this seems to be a SSR issue (maybe because it's a test? idk). I tried this solution out and it worked. The error I was getting was:
![image](https://user-images.githubusercontent.com/5455859/39025063-ab9a2b94-43f9-11e8-946d-289d23ae5f36.png)
I found [this issue](https://github.com/apollographql/subscriptions-transport-ws/issues/87) that's pretty similar, but the implementations they're using are different. I can do some more digging, but I have a feeling I'd have to change our implementation/it might take longer than desired to do this a different way.

Reply:
Basically the library isn't getting the `WebSocket` library it needs for some reason. I saw some other people say they tried making it `global`, but that still didn't work.

Reply:
1. 👍 
2. My main concern is that prod Electron version might be different for `process.browser` (may be undefined). So if `process.browser` is undefined in the Electron prod version the WS won't work. Would like to hold merging this PR until after mainnet release. Code looks fine to me, but just want to make sure nothing breaks right before our launch.

Reply:
Electron is not your typical environment. Like for example, `process.env` was not able to be found in the prod version. Something with how Electron's main/renderer processes work.

Reply:
makes total sense. I'll check that out tomorrow